app/javascript/stylesheets/_comp-headings_blocks.scss

```diff
@@ -39,7 +39,7 @@ h4 {
   }
 
   .heading_breadcrumb::after {
-    font-family: "Font Awesome 5 Pro";
+    font-family: "Font Awesome 5 Pro", sans-serif;
```

Comment:
Question and doubt, if we are using the free version of font-awesome in the package.json, will the PRO version work?

`"@fortawesome/fontawesome-free": "^5.15.4",`

Reply:
Nice point. I didn't realise it when the linter enforced me to provide an alternative font.

`.heading_breadcrumb` is a very old class (before my days) so it's likely gobierto had FA-pro back then. I'll change it properly, as now it will fallback to _sans-serif_ always

Reply:
Updated